pkg/server/interceptors.go

```diff
@@ -5,17 +5,21 @@ package server
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
 	"io"
 
 	"go.uber.org/zap"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/peer"
 	"google.golang.org/grpc/status"
 
+	"storj.io/storj/pkg/identity"
 	"storj.io/storj/storage"
 )
 
-func streamInterceptor(srv interface{}, ss grpc.ServerStream, info *grpc.StreamServerInfo, handler grpc.StreamHandler) (err error) {
+func logOnErrorStreamInterceptor(srv interface{}, ss grpc.ServerStream, info *grpc.StreamServerInfo, handler grpc.StreamHandler) (err error) {
```

Comment:
Why change the name vs add a comment?

Reply:
Just because the name becomes confusing once there are other StreamInterceptor instances around.